rust/core-lib/src/view.rs

```diff
@@ -980,13 +1005,30 @@ impl View {
     /// indicates a search for the next occurrence past the end of the file.
     pub fn select_next_occurrence(&mut self, text: &Rope, reverse: bool, wrapped: bool,
                                   _allow_same: bool, modify_selection: &SelectionModifier) {
+        let (cur_start, cur_end) = match self.selection.last() {
```

Comment:
I might choose to write this,

```rust 
... self.selection.last().map(|s| (s.min(), s.max()).unwrap_or((0, 0))
```

but it's purely a style thing, and it's totally reasonable as written.

Reply:
Personally, I find it as is easier to read. I would leave it unchanged